requirements.txt

```diff
@@ -24,7 +24,7 @@ simplejson
 six
 sqlalchemy
 sqlalchemy_utils
-threddsclient==0.3.4
+threddsclient==0.3.5
```

Comment:
Add `FIXME` note corresponding to Dockerfile `request` installation order when updated with fix mentioned.